api/index.ts

```diff
@@ -17,7 +17,11 @@ app.use('/doc', swaggerUi.serve, swaggerUi.setup(swaggerFile))
 app.use("/countries", express.static(__dirname + "/public/countries/countries.json"));
 /* Endpoints */
 require('./endpoints/auth')(app)
-require('./endpoints/postgres')(app)
+require('./endpoints/emissions')(app)
+require('./endpoints/groups')(app)
+require('./endpoints/schools')(app)
+require('./endpoints/sessions')(app)
 require('./endpoints/exports')(app)
+require('./endpoints/languages')(app)
```

Comment:
Pourquoi avori mis endpoint en debut dechaque route ? Est-ce qu'on va avoir des route sans endpoints ?

Reply:
C'est plutôt pour avoir un fichier endpoints et un models par exemple mais je peux mettre index.ts dans endpoints c'est peut être plus logique

Reply:
Je ne comprends pas bien ton commentaire, on peut peut être s'appeler à ce sujet ?

Reply:
C'est vrai qu'une structure de ce type pourrait être plus intéressante https://dev.to/ericlecodeur/nodejs-express-partie-5-routes-et-controllers-18l ?

Reply:
Oui d'habitude je fais une structure un peu dans ce type. Mais ici plus que la structure des fichiers c'est le fait que toutes les routes commencent par endpoint je ne vois pas trop à quoi ca sert (bien que ce ne soit pas gênant)

Reply:
C'est pas vraiment les routes qui commence par ça mais juste le path du fichier